defs/bugs/storage/ceph.yaml

```diff
@@ -0,0 +1,18 @@
+38745:
+  requires:
+    and:
+      - apt:
+          ceph-osd:
+            - min: 15.2.6
+              max: 15.2.10
+    or:
+      - property: core.plugins.storage.ceph.CephDaemonConfigShowAllOSDs.bluestore_volume_selection_policy
+        value: [use_some_extra]
+        operator: ne
```

Comment:
As per documentation, the tag should be op instead of operator
https://github.com/canonical/hotsos/tree/master/defs#requires